recipes/mad/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+
+if [[ $(uname) == Darwin ]]; then
+    export CC=clang
```

Comment:
Why are you using `clang` instead of `$CC` as set by conda-build here?

Reply:
copied an old recipe and modified it. apparently it was needed at some point, but no longer necessary.
I've deleted that line in the latest commit